src/embedded.js

```diff
@@ -98,22 +95,16 @@ class Embedded {
       this.ui.registerModule(AuthModule, { auth: null })
     }
     // Register UI modules
-    this.ui.registerModule(PanelModule, {
-      mountPoint: '#alpheios-panel-embedded'
```

Comment:
will the elements not get ids anymore? we used to use different ids for the webextension and embed-lib to avoid conflicts. Is this going to be an issue?

Reply:
Before we were mounting all elements into the parent container that was a part of an HTML template. We don't have this template any more. I'm not sure if we need to distinguish between webextension and embed-lib now because they should not be active simultaneously so only one or the other be active at a time. But if we do need to keep them separate, there are two options:

1. We can manually insert a mounting container for the embed-lib elements in the the HTML, either by editing an HTML code of the host page or by inserting it dynamically into the page with JS. Each module has a `mountTo` option. Using that, we can specify that they need to be mounted to the specific container element.
2. Each module has a `rootElementId` which, when set, will assign its value to the id attribute of the root element of the component (i.e. panel, popup, etc.). With this, we can set custom id for each specific component. However, while being in the module code, it's used in a template only for action panel at the moment. It's more there for future extensibility now. But I can update the template code to use it for all components if needed.

So please let me know if we need those options, and if we do, which ones are required (or maybe both). I will then adjust the code and test it to make sure everything works fine.

Reply:
I am not sure. I think preventing activation of the extension on an embedded library site should be sufficient for the time being. Let's just test after you commit these changes and see if further work is needed.

Reply:
I did a test of both webextension and embed-lib together on the page, and they were working fine for me. Webextension inserted it's own elements and embed-lib - it's own. The coexist together on the page. This might be a problem as pairs of elements have the same ids, but because elements of webextension are hidden with `display; none` that's fine, I think. 
Probably we can test it after merging and, if any problems are still found, make amends.